docs/how-to/how-to-use-virtio-fs-nydus-with-kata.md

```diff
@@ -16,15 +16,15 @@ You can use Kata Containers with `nydus` as follows,
 
 4. Use [kata-containers](https://github.com/kata-containers/kata-containers) `latest` branch to compile and build `kata-containers.img`;
 
-5. Update `configuration-qemu.toml` to include:
+5. Update `configuration-qemu.toml` or `configuration-qemu.toml`to include:
 
 ```toml
 shared_fs = "virtio-fs-nydus"
 virtio_fs_daemon = "<nydusd binary path>"
 virtio_fs_extra_args = []
 ```
 
-6. run `crictl run -r kata-qemu nydus-container.yaml nydus-sandbox.yaml`;
+6. run `crictl run -r kata-qemu/kata-clh nydus-container.yaml nydus-sandbox.yaml`;
```

Comment:
I think `kata-qemu/kata-clh` can simply change to `kata`.

Reply:
agree, have updated